android/src/main/kotlin/com/hypertrack/sdk/flutter/FlutterSerialization.kt

```diff
@@ -0,0 +1,52 @@
+package com.hypertrack.sdk.flutter
+
+import com.hypertrack.sdk.flutter.common.*
+import io.flutter.plugin.common.MethodCall
+import io.flutter.plugin.common.MethodChannel
+import io.flutter.plugin.common.EventChannel.EventSink
+
+private const val ERROR_CODE_METHOD_CALL = "method_call_error"
```

Comment:
Shouldn't that go to the `companion object` that is placed in the `HyperTrackPlugin.kt` to be in one place?

Reply:
fixed